tsconfig.json

```diff
@@ -24,6 +25,6 @@
         },
         "types": ["cypress", "cypress-real-events", "vite/client"]
     },
-    "include": ["src/types", "src/**/*"],
+    "include": ["src/types", "src/**/*", "cypress/support/component.ts"],
```

Comment:
why does component.ts need to be included in the output build?

Reply:
in the component the typing for the `mount` command is defined. That the test file know of this `cy.mount` it has to be included there. Would it be better/clearer if I would make a file `cypress/support/types.ts`?

Reply:
Nono i was just curious to understand the include :) Thx for clarifying :)